freqtrade/exchange/exchange.py

```diff
@@ -1964,6 +1964,37 @@ def get_funding_fees(self, pair: str, amount: float, open_date: datetime) -> flo
         else:
             return 0.0
 
+    @retrier
+    def get_liquidation_price(self, pair: str):
+        '''
+            Set's the margin mode on the exchange to cross or isolated for a specific pair
+            :param pair: base/quote currency pair (e.g. "ADA/USDT")
+        '''
+        if self._config['dry_run'] or not self.exchange_has("fetchPositions"):
+            # Some exchanges only support one collateral type
+            return
+
+        try:
+            return self._api.fetch_positions(pair).liquidationPrice
+        except ccxt.DDoSProtection as e:
+            raise DDosProtection(e) from e
+        except (ccxt.NetworkError, ccxt.ExchangeError) as e:
+            raise TemporaryError(
+                f'Could not set margin mode due to {e.__class__.__name__}. Message: {e}') from e
+        except ccxt.BaseError as e:
+            raise OperationalException(e) from e
+
+    def get_maintenance_ratio_and_amt(
+        self,
+        pair: str,
+        nominal_value: Optional[float] = 0.0,
+    ):
```

Comment:
we should/must have the return Typed here.

As we expect to override this - having proper types will have mypy "scream" at us if we return something unexpected.